reactioncommerce/reaction-next-starterkit
# refactor: nnnnat checkout login

Resolves #248 
Impact: **minor**
Type: **refactor**

## Issue
When entering checkout as an anonymous customer, you'd sometimes see a flash of "checkout" view before the login/register "entry" view would be displayed. 

## Solution
* Move login/register/guest entry out of the `login` & `checkout` pages and to its own component `Entry`
* Updated `login` & `checkout` pages to use the `Entry` component.
* Created a cart spacific login route ("/cart/login") to keep the curret `login` page working as is.

## Breaking changes
N/A

## Testing
1. As an anonymous customer add a product to your cart and entry checkout.
2. The page should redirect to the login page without showing a flash of the checkout.
3. click the back button to go back to the last page before trying to entry checkout.
4. Enter checkout and fill out the guest form.
5. The page should redirect to "/cart/checkout" and the checkout action view should render.
